### DISCUSSION TITLE
Where /How to view all shared folders

### DISCUSSION THREAD

**Discussion by shinemoon** [photoprism/photoprism#3961] [Q&A] [answered]
Hi,

Though I know how to share the folder (with the link and works), it's a bit confused for me how to find all my shared folders? Especially when I have so many folders/sub-folders, I just forget which were exactly shared ... 

Is there some tab/place I can soon identify which folders have been shared by me? 

Thanks!

**Comment by graciousgrey** [comment, marked as answer]
A section that shows all shared albums/folders is planned but not implemented yet. You can recognise albums/folders that are shared by the sharing icon in the upper left corner.

**Reply by shinemoon** [reply]
Great thanks, I can live with that and of course hope feature can be in place soon.
